src/middleware/isAuth.js

```diff
@@ -17,11 +17,16 @@ async function isAuth(req, res, next) {
           username: req.session.passport.user.username,
         });
         break;
-      default:
+      case "google":
         authorizeCondition = await Users.findOne({
           userId: req.session.passport.user.id,
           email: req.session.passport.user.emails[0].value,
         });
+      case "default":
```

Comment:
Even if you named your provider as "default", if the provider will not be github or google, it will take the default case from the switch, so you can go with:

```suggestion
      default:
```